feature: adding subdomain center as source

iamshreeram opened the pull request:
This source returns the empty list for almost all the domains. The original [sourcecode](https://github.com/owasp-amass/amass/blob/5f1f7176bae60975c1e5be64273cb201f1bb37c3/resources/scripts/api/subdomaincenter.ads) has rate limiting (upto 2 request) as well.

edoardottt commented on pkg/opendb/subdomaincenter.go:
```diff
@@ -0,0 +1,70 @@
+// SubdomainCenter retrieves from the url below some known subdomains.
+func SubdomainCenter(domain string, plain bool) []string {
+	if !plain {
+		fmt.Println("Pulling data from ThreatCrowd")
```
It's not threatcrowd

edoardottt commented on pkg/opendb/subdomaincenter.go:
```diff
@@ -0,0 +1,70 @@
+	defer resp.Body.Close()
+
+	dec := json.NewDecoder(resp.Body)
+	err = dec.Decode(&wrapper)
```
Here trying this code I get the err `json: cannot unmarshal array into Go value of type struct { Records []string "json:\"subdomains\"" }`

iamshreeram commented:
@edoardottt , Please look into [5217aa6](https://github.com/edoardottt/scilla/pull/118/commits/5217aa68f909e3c66f0a3061900a4b1bbdb9447c)

edoardottt commented on pkg/opendb/subdomaincenter.go:
```diff
@@ -0,0 +1,74 @@
+// SubdomainCenter retrieves from the url below some known subdomains.
+func SubdomainCenter(domain string, plain bool) []string {
+	if !plain {
+		fmt.Println("Pulling data from Subdomain")
```
Should be `Pulling data from Subdomain Center`

iamshreeram commented:
1. Sure. Thanks!
2. I have added the code to main enumeration, both in `ReportSubcommandHandler` and `SubdomainSubcommandHandler`
3. I have fixed the linting problem. 

Please review - #[df98dcc](https://github.com/edoardottt/scilla/pull/118/commits/df98dccc464a9add624e895720dc09e432b75ba4)

edoardottt reviewed (approved):
lgtm
